cli/command/container/update.go

```diff
@@ -66,6 +67,7 @@ func NewUpdateCommand(dockerCli *command.DockerCli) *cobra.Command {
 	flags.Var(&opts.memorySwap, "memory-swap", "Swap limit equal to memory plus swap: '-1' to enable unlimited swap")
 	flags.Var(&opts.kernelMemory, "kernel-memory", "Kernel memory limit")
 	flags.StringVar(&opts.restartPolicy, "restart", "", "Restart policy to apply when a container exits")
+	flags.Int64Var(&opts.pidsLimit, "pids-limit", 0, "Tune container pids limit (set -1 for unlimited)")
```

Comment:
Before I forget to comment; this needs an "annotation" so that the flag is hidden when connecting to an older daemon version;

```go
flags.SetAnnotation("cpus", "version", []string{"1.30"})
```